python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--inputfilePath', type=str, default='', help='ranklib input file path')
+parser.add_argument('--outputfilePath', type=str, default='', help='output converted file path')
+parser.add_argument('--keepAdditionalInfo', type=int, default=1, help='Option to keep additional info (All info after the "#") 1 to keep, 0 to ignore')
+parser.add_argument('--GL2Clicks', type=int, default=1, help='Convert graded relevance to clicks (only max relevant document is considered clicked) 1 to convert')
+args = parser.parse_args()
+query_id_name = 'qid'
```

Comment:
Consider moving these two (`query_id_name` and `relevance_name`) below the imports and upper-casing them; they seem user choices and this would enable other users to easily change them.